src/capture.c

```diff
@@ -66,17 +66,12 @@ static bool get_exe(char *buf, size_t bufsize)
 
 static bool capture_try_connect()
 {
-    const char *sockname = "/tmp/obs-vkcapture.sock";
-
-    if (access(sockname, R_OK) != 0) {
-        return false;
-    }
-
     struct sockaddr_un addr;
-    addr.sun_family = AF_UNIX;
-    strcpy(addr.sun_path, sockname);
-    int sock = socket(AF_UNIX, SOCK_STREAM | SOCK_CLOEXEC | SOCK_NONBLOCK, 0);
-    int ret = connect(sock, (const struct sockaddr *)&addr, sizeof(addr));
+    addr.sun_family = PF_LOCAL;
+    const int namesize = snprintf(addr.sun_path, sizeof(addr.sun_path), "%c%s", '\0', SOCKNAME);
+
+    int sock = socket(PF_LOCAL, SOCK_STREAM | SOCK_CLOEXEC | SOCK_NONBLOCK, 0);
+    int ret = connect(sock, (const struct sockaddr *)&addr, sizeof(addr.sun_family) + namesize);
```

Comment:
`sizeof(addr)` still works

Reply:
It would make the socket appear as `@/com/obsproject/vkcapture@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@` (the array size is 108 bytes) because the NUL character does not mark the end of the string.